.ci/E2E-tests/execute.sh

```diff
@@ -1,18 +1,41 @@
 #!/bin/sh
 
-DB=$1
+PIPELINE=$1
+TEST_FRAMEWORK=$2
 
-if [ "$DB" = "mysql" ]; then
-  COMPOSE_FILE="cypress-E2E-tests-mysql.yml"
-elif [ "$DB" = "postgres" ]; then
-  COMPOSE_FILE="cypress-E2E-tests-postgres.yml"
-elif [ "$DB" = "local" ]; then
-  COMPOSE_FILE="cypress-E2E-tests-local.yml"
+echo "Pipeline:"
+echo "$PIPELINE"
+echo "Test framework:"
+echo "$TEST_FRAMEWORK"
+
+if [ "$TEST_FRAMEWORK" = "playwright" ]; then
+  if [ "$PIPELINE" = "mysql" ]; then
+    COMPOSE_FILE="playwright-E2E-tests-mysql.yml"
+  elif [ "$PIPELINE" = "local" ]; then
+    COMPOSE_FILE="playwright-E2E-tests-local.yml"
+  else
+      echo "Invalid database type. Please choose among mysql, postgres or local."
```

Comment:
I think calling this `PIPELINE` is not really accurate.
- "local" is not a database type, but a mysql with LocalVC/CI
- "mysql" and "postgres" however are only a distinction between which database to use
- Ultimately this var decides which tech stack to use. It does not really change what steps are executed in a job/pipeline.

I think you should rename this var to better fit its purpose and rename "local" to "mysql-localci", so that it is very clear what is meant.